adminSiteClient/GdocsSettingsForm.tsx

```diff
@@ -22,17 +22,13 @@ export const GdocsSettingsForm = ({
     setGdoc: (gdoc: OwidGdocInterface) => void
     errors?: OwidGdocErrorMessage[]
 }) => {
-    const attachmentMessages = errors?.filter((error) =>
-        ["linkedDocuments", "imageMetadata", "content"].includes(error.property)
+    // These errors don't have a specific form field to render them in. We just show them at the bottom of the drawer
```

Comment:
Any reason not to have `type` as a visual field so we could surface related errors the same way we surface errors in other fields (red border, etc...)?